README.md

```diff
@@ -13,10 +13,89 @@ Open Distro for Elasticsearch Security Parent is a parent repo for the Open Dist
 ## Documentation
 Please refer to the [technical documentation](https://opendistro.github.io/for-elasticsearch-docs) for detailed information.
 
+## Build
+To build the security plugin from source follow these instructions **in this order**:
+
+
+Download the `security-parent` source code
+
+`git clone https://github.com/opendistro-for-elasticsearch/security-parent.git`
+
+`cd security-parent`
+
+
+Install to the .m2 directory: 
+
+
+`mvn clean install`
+
+`cd ..`
+
+Download the `security` source code
+
+`git clone https://github.com/opendistro-for-elasticsearch/security.git`
+
+`cd security`
+
+Install to the `.m2` directory: 
+
+
+`mvn clean install`
+
+`cd ..`
+
+Download the `security-ssl` source code **(only for versions < 1.0.0.0 [ Elasticsearch 7 ])**
+
+`git clone https://github.com/opendistro-for-elasticsearch/security-ssl.git`
+
+Install to the `.m2` directory: 
+
+
+`cd security-ssl`
+
+`mvn clean install`
+
+`cd .. `
+
+
+Download the `security-advanced-modules` source code
+
+`git clone https://github.com/opendistro-for-elasticsearch/security-advanced-modules.git`
+
+Install to the `.m2` directory: 
+
+
+`cd security-advanced-modules`
+
+`mvn clean install`
+
+`cd .. `
+
+Build the Elasticsearch plugin
+
+`cd security`
+
```

Comment:
I would change the following two line to

`mvn clean package -Padvanced` 
The above builds the final artifacts in zip format. 

The artifacts can be found in:
 `cd target/releases/`